### ISSUE TITLE
iHAMOCC: convert to SI units (CGS -> MKS)

### ISSUE THREAD

**Issue by TomasTorsvik** [NorESMhub/BLOM#223]
BLOM is introducing SI units (#219). We need to make adjustments in iHAMOCC accordingly.

**Comment by jmaerz** [comment]
Hi @TomasTorsvik , now that the SI unit conversion (now MKS [meter, kg, seconds] instead of CGS [cm, gram, seconds]) has been merged into master, I'll also have a look into it (also due to some further dependencies in my branch). If I remember correctly from the BLOM-internal discussion, we only (mainly?) have to touch the wrapper function. Looks, as if the version is controlled via pre-processor flags `MKS` or `CGS`, which we can use in the iHAMOCC wrapper as well.

**Comment by TomasTorsvik** [comment]
Hi @jmaerz , yes I think it should be possible to solve this problem in the wrapper function. I will not have time to look at this issue this week, or at least, not until Friday.

**Comment by jmaerz** [comment]
Hi @TomasTorsvik , I will locally branch off from `master` and will have a look into it in the afternoon. I'll let you know, how far I got.
